src/modules/10-common/RouteDefinitions.ts

```diff
@@ -1898,6 +1898,17 @@ const routes = {
       environmentIdentifier
     }: Partial<ProjectPathProps> & { environmentIdentifier: string }) =>
       `/chaos/orgs/${orgIdentifier}/projects/${projectIdentifier}/environments/${environmentIdentifier}`
+  ),
+
+  /*********************** IACM */
+  toIacm: withAccountId(() => `/iacm`),
+  toIacmMicroFrontend: withAccountId(
```

Comment:
Remember IACM is an initialism, not a word, so each letter should remain uppercase when camel-cased.
```suggestion
  toIACM: withAccountId(() => `/iacm`),
  toIACMMicroFrontend: withAccountId(
```